.goreleaser.yml

```diff
@@ -147,6 +147,7 @@ nfpms:
     priority: extra
     scripts:
       postinstall: .ci/goreleaser/postinstall.sh
+      preremove: .ci/goreleaser/preremove.sh
```

Comment:
As this patch set applies to [`deb` and `rpm` formats](https://github.com/siemens/wfx/blob/main/.goreleaser.yml#L143-L144), how is the convention on RPM-based distros, e.g., Fedora, RHEL, ... regarding starting a service after installation? If that differs from Debian's, we need to switch-case here.

Reply:
> Further, it seems only wfx is built for deb and rpm, but not wfxctl and wfx-viewer. Is this intentional?

These are released in extra packages, see, e.g., the release artifacts for [v0.4.1](https://github.com/siemens/wfx/releases/tag/v0.4.1). This way, one can decide fine-grained what's needed or not for the application scenario at hand.